src/library-authoring/LibraryAuthoringPage.test.tsx

```diff
@@ -189,33 +189,33 @@ describe('<LibraryAuthoringPage />', () => {
     axiosMock.onGet(getContentLibraryApiUrl(libraryData.id)).reply(200, libraryData);
 
     const {
-      getByRole, getByText, queryByText, findByText, findAllByText,
+      getByRole, getAllByText, getByText, queryByText, findByText, findAllByText,
     } = render(<RootWrapper />);
 
-    // Ensure the search endpoint is called:
-    // Call 1: To fetch searchable/filterable/sortable library data
-    // Call 2: To fetch the recently modified components only
-    await waitFor(() => { expect(fetchMock).toHaveFetchedTimes(2, searchEndpoint, 'post'); });
+    await waitFor(() => { expect(fetchMock).toHaveFetchedTimes(1, searchEndpoint, 'post'); });
```

Comment:
@pomegranited One question: what changes here to decrease the number of calls of the searchEndpoint?

Reply:
Honestly, I wish I knew for sure.. Maybe it's something to do with the default sort option not being empty anymore (since it's `created:desc` now), and maybe [the search manager was setting it twice before for some reason](https://github.com/openedx/frontend-app-course-authoring/pull/1222/files#diff-6f08172263aa729980f7d9c562d1d76b36d7680cc75e492d5df35d289d0a60d0R69-R73)?

But the premise in those deleted comments is not correct -- yes, we make 2 queries, but they're done as part of a single hit to the `/multi-search` endpoint. So it should have been 1 hit all along.

That's my theory, anyway.